packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -130,28 +130,30 @@ export type InputBoxButtonProps = {
   onClick: (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
   className?: string;
   id?: string;
+  ariaLabel?: string;
 };
 
 /**
  * @private
  */
 export const InputBoxButton = (props: InputBoxButtonProps): JSX.Element => {
-  const { onRenderIcon, onClick, className, id } = props;
-  const [isMouseOverSendIcon, setIsMouseOverSendIcon] = useState(false);
+  const { onRenderIcon, onClick, ariaLabel, className, id } = props;
+  const [isMouseOverIcon, setIsMouseOverIcon] = useState(false);
```

Comment:
nit.

Something feels very wrong about us holding onto the state that tells us if we are hovering over the icon.
If this makes sense to everyone else tho then lets land it

Reply:
Agree, but was part of component before, so I did not want to touch that. Now that we want the props to be an extension of IButtonProps, I have to revisit this

Reply:
the issue here is that we have a complete different icon when overring

Reply:
left a question to FluentUI team. Will follow up